### PR TITLE
Full Calendar

### PR DESCRIPTION
**Pull request by ChristianMurphy**
Alternative to #36 

*TODO* needs [event data](http://fullcalendar.io/docs/event_data/events_array/)
@sylvr3

### CODE REVIEW

**Comment by sylvr3** [comment]
@ChristianMurphy I think FullCalendar is a good choice. :+1:  I made some hardcoded events, just trying to get them to read it from the JSON object.

**Comment by sylvr3** [comment]
@ChristianMurphy I'm having trouble parsing the JSON object without it breaking the calendar code. JSON.parse() will not work with two or three curly braces (ie JSON.parse('{{{ surveys }}}');

**Comment by ChristianMurphy** [comment]
Make sure to `JSON.stringify` the Object/Array in the presenter.
Then in the template/view `JSON.parse` the Object/Array back out.

**Comment by ChristianMurphy** [comment]
Also always use triple curlies `{{{ }}}` when sending raw JSON or HTML

File: presenter/patient.js [hunk 1]
```diff
@@ -25,37 +25,37 @@ module.exports = function (request, reply) {
             id: 1,
             name: 'test'
         },
-        surveys: [
+        surveys: JSON.stringify([
```
Comment (sylvr3) [inline presenter/patient.js]: @ChristianMurphy Am I stringifying the surveys JSON array correctly?

**Comment by ChristianMurphy** [comment]
That is the correct format for the calendar, the table needs the plain JS array rather than a stringified version.

**Comment by ChristianMurphy** [comment]
calendar

**Comment by sylvr3** [comment]
Thanks for adding those additional attributes to the events @ChristianMurphy
Don't we want the completed surveys to be in green?

**Comment by ChristianMurphy** [comment]
sure

**Comment by sylvr3** [comment]
And shouldn't we make the events repeat once a week for a certain interval (for instance, if it is a 3 week trial, we can make a recurring event on every Monday til the end date?)
For a monthly trial, on the calendar shouldn't we be showing it being scheduled once a month for every month til the end of the trial?

**Comment by ChristianMurphy** [comment]
No, Schedule is a concept that does not affect Surveys.
Schedule is used to generate surveys daily.
Each patient will be offered only a single survey each day.
When multiple types of survey fall on the same day (monthly, weekly daily).
The least frequently offered survey would be generated (monthly).

Showing survey span could be interesting to show in a Stage view, where a clinician can configure how surveys are generated.
But we will cross that bridge when we come to it.

**Comment by sylvr3** [comment]
:+1:

**Pull request merged**